adapters/openx/openx.go

```diff
@@ -169,6 +169,11 @@ func (a *OpenxAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalReq
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
 
+	// overrride default currency
+	if bidResp.Cur != "" && bidResp.Cur != "USD" {
```

Comment:
Is the check for `bidResp.Cur != "USD"` necessary? I know it's currently the default, but is it worth to hardcode that assumption here?

Reply:
not really, will remove it

Reply:
done